test/e2e/bootstrapper_test.go

```diff
@@ -26,24 +30,48 @@ func init() {
 	pflag.StringVar(&installDir, "install-dir", "C:\\k", "Installation directory")
 }
 
-// This test will only test whether or not the bootstrapper was able to start the required services. It gives no
-// guarentees that the underlying executables (kubelet) are functioning as they should.
+// TestBootstrapper tests that the bootstrapper was able to start the required services
 // TODO: Consider adding functionality to this test to check if the underlying processes are running properly,
 //  	 otherwise keep that functionality contained within other future tests
 func TestBootstrapper(t *testing.T) {
+	var kubeletRunningOnStart bool
```

Comment:
The name is a bit confusing considering what you're doing here - https://github.com/openshift/windows-machine-config-operator/pull/54/files#diff-564dc47aef3986957382c41ba37de57fR44, do you want to be explicit on what this means? smthing which signifies if kubelet is running before starting of test.

Reply:
Good idea, will implement